creator-cards/ricardo-perez.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ricardo Josue Perez Altamirano",
+  "type": "CREATOR",
+  "tags": ".NET, C#, ASP.NET, I LOVE DOTNET,Azure",
```

Comment:
Could you remove I LOVE DOTNET is was only sample data and not meant as a real tag

Reply:
yes of couse, the changes is in commit c3584742fd901d3adafc53afc7b136eb96576509